jsk_pcl_ros_utils/src/bounding_box_array_to_bounding_box_nodelet.cpp

```diff
@@ -81,7 +81,10 @@ namespace jsk_pcl_ros_utils
     bbox_msg.header = bbox_array_msg->header;
 
     int array_size = bbox_array_msg->boxes.size();
-    if (index_ < 0) {
+    if (array_size == 0)
+    {
```

Comment:
Adding the `NODELET_WARN_ONCE` message to notify that the subscribed empty bounding box array is better.

Reply:
I update to add NODELET_WARN